aten.bzl

```diff
@@ -3,7 +3,7 @@ load("@rules_cc//cc:defs.bzl", "cc_library")
 
 CPU_CAPABILITY_NAMES = ["DEFAULT", "AVX2"]
 CAPABILITY_COMPILER_FLAGS = {
-    "AVX2": ["-mavx2", "-mfma"],
+    "AVX2": ["-mavx2", "-mfma", "-mf16c"],
```

Comment:
Can you please add some reference to the doc that `-mfc16c` is always available whenever `-mavx2` is is available?

Reply:
Oh this is really important, i should add more comment. I have parsed the gcc compilation flag with **-march** from https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html

**FP16C** is added after **AVX**, but before **AVX2**. If the hardware has **AVX2**, it has **FP16C**. I have listed the detailed info in a table below.